x11-misc/lightdm/lightdm-1.32.0-r3.ebuild

```diff
@@ -120,7 +116,6 @@ src_configure() {
 		--disable-tests
 		$(use_enable audit libaudit)
 		$(use_enable introspection)
-		$(use_enable qt5 liblightdm-qt5)
```

Comment:
You're introducing automagic here:
`AS_HELP_STRING([--enable-liblightdm-qt5],[Enable LightDM client Qt5 libraries [[default=auto]]]),`

With Qt5 installed, even if you don't want it, now this happens in configure:
```
checking for
        Qt5Core
        Qt5DBus
        Qt5Gui
    ... yes
...
                    Light Display Manager 1.32.0
                    ===========================
...
        liblightdm-qt5:           yes
```